processFlows.json

```diff
@@ -1546,5 +1546,519 @@
+          }
+        }
+      },
+      { "op": "And" },
+      {
+        "restriction": {
+          "MatchInputOutputMetadataValue": {
+            "inputIndex": 2,
+            "inputMetadataKey": "demandA",
+            "outputIndex": 2,
+            "outputMetadataKey": "demandA"
+          }
+        }
+      },
+      { "op": "And" },
+      {
+        "restriction": {
+          "MatchInputOutputMetadataValue": {
+            "inputIndex": 1,
+            "inputMetadataKey": "originalId",
+            "outputIndex": 2,
+            "outputMetadataKey": "demandB"
+          }
+        }
+      },
+      { "op": "And" },
+      {
+        "restriction": {
+          "MatchInputOutputMetadataValue": {
+            "inputIndex": 2,
+            "inputMetadataKey": "demandB",
+            "outputIndex": 2,
+            "outputMetadataKey": "demandB"
+          }
+        }
+      },
+      { "op": "And" },
+      {
+        "restriction": {
+          "MatchInputOutputMetadataValue": {
+            "inputIndex": 2,
+            "inputMetadataKey": "originalId",
+            "outputIndex": 2,
+            "outputMetadataKey": "originalId"
+          }
+        }
+      },
+      {
+        "restriction": {
+          "InputHasMetadata": {
+            "index": 2,
+            "metadataKey": "originalId"
+          }
+        }
+      },
+      {
+        "restriction": {
+          "None": {}
+        }
+      },
+      { "op": "NotL" },
+      {
+        "restriction": {
+          "MatchInputIdOutputMetadataValue": {
+            "inputIndex": 2,
+            "outputIndex": 2,
+            "outputMetadataKey": "originalId"
+          }
+        }
+      },
+      { "op": "And" },
+      { "op": "Or" },
+      { "op": "And" },
+      {
+        "restriction": {
+          "FixedInputMetadataValue": {
+            "index": 2,
+            "metadataKey": "state",
+            "metadataValue": {
+              "Literal": "acceptedFinal"
+            }
+          }
+        }
+      },
+      { "op": "And" },
+      {
+        "restriction": {
+          "FixedOutputMetadataValue": {
+            "index": 2,
+            "metadataKey": "state",
+            "metadataValue": {
+              "Literal": "cancelled"
+            }
+          }
+        }
+      },
+      { "op": "And" },
+      {
+        "restriction": {
+          "SenderHasOutputRole": {
+            "index": 2,
+            "roleKey": "MemberA"
+          }
+        }
+      },
+      {
+        "restriction": {
+          "SenderHasOutputRole": {
+            "index": 2,
+            "roleKey": "MemberB"
+          }
+        }
+      },
+      { "op": "Or" },
+      { "op": "And" },
+      {
+        "restriction": {
+          "FixedOutputMetadataValueType": {
+            "index": 2,
+            "metadataKey": "comment",
+            "metadataValueType": "File"
+          }
+        }
+      },
+      { "op": "And" },
+      { "op": "Or" },
+      { "op": "And" }
```

Comment:
The last 2 op entries here are likely invalid. Have you tried running this into the node? You can do that with:

```sh
docker compose up -d
```

and then

```sh
npm run flows
```

Reply:
![Screenshot 2023-09-14 at 09 13 44](https://github.com/digicatapult/dscp-matchmaker-api/assets/58051865/c551c54f-b4aa-44f1-a665-d6653023c988)
That worked - I was actually going to ask about the last two lines this morning.